net/apfree-wifidog/Makefile

```diff
@@ -27,7 +27,7 @@ define Package/apfree-wifidog
   SUBMENU:=Captive Portals
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+zlib +libjson-c +libevent2 +libevent2-openssl +libuci +openssl-util +libnetfilter-queue
+  DEPENDS:=+zlib +libjson-c +libevent2 +libevent2-openssl +libuci +openssl-util +libnetfilter-queue +conntrack +libmosquitto
```

Comment:
I guess these are part of the update, but please mention them in the commit message.

Reply:
@1715173329 
Conntrack was utilized to fix a bug in Apfree-Wifidog, where a user's long-lived connection persisted even after logging out.
Libmosquitto was integrated as a preparatory step toward future MQTT protocol support, though the implementation is not yet complete.
For more detailed information, please refer to the release notes included in this PR.
https://github.com/liudf0716/apfree-wifidog/releases/tag/7.10.2082

Thank you for the thorough code review!

Reply:
Please mention these changes in commit message, thanks.